doc/nrf/protocols/matter/index.rst

```diff
@@ -54,7 +56,7 @@ The |NCS| allows you to develop applications with different versions of Matter,
    These changes are listed in the Matter fork section of the |NCS| :ref:`release_notes`.
 
 .. important::
-   For Matter over Thread samples, starting with |NCS| releases after v2.5.2, the default cryptography backend is Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
+   For Matter over Thread samples, starting with |NCS| releases after v2.6.0, the default cryptography backend is Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
```

Comment:
```suggestion
   For Matter over Thread samples, starting from |NCS| v2.6.0, the default cryptography backend is Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
```
after v2.6.0 isn't right.

Reply:
Mbed TLS is used as a vague construct in this case, and in fact the most common source for PSA crypto APIs implemented (in software) is through the Mbed TLS deliverables.

I recommend we change the Mbed TLS to `m̀bedtls_`-prefixed crypto toolbox APIs from Mbed TLS